### ISSUE TITLE
[Eclipse IDE] Do not use StyledText as JAWS does not read well

### ISSUE THREAD

**Issue by eharris369** [eclipse/codewind#1696]
JAWS does not appear to work well with StyledText.  Adding the following will get it to read the styled text when the user tabs to it but when the user does an Insert (JAWS key) + B to read the entire dialog it skips it:
```
		typeText.getAccessible().addAccessibleListener(new AccessibleAdapter() {
			@Override
			public void getName(AccessibleEvent e) {
				e.result = typeText.getText();
			}
		});
```
It would be better to replace StyledText with Text and use bold font when needed.

**Comment by eharris369** [comment]
Fixed by PR: https://github.com/eclipse/codewind-eclipse/pull/509
